mailpoet/assets/js/src/public.tsx

```diff
@@ -536,10 +536,13 @@ jQuery(($) => {
       );
 
       let registerForm: JQuery<HTMLFormElement>;
+      let submitInput: object;
       if (wpRegisterForm.length) {
         registerForm = wpRegisterForm;
+        submitInput = { type: 'hidden', name: 'wp-submit', value: 'Register' };
```

Comment:
Just curious, does the value matter in this case? Because if it does, it would require translation.

Reply:
No. This value is rendered by the WP/WC form itself and we should not tamper with it. The backed logic of WP/WC expect it as is.

Reply:
I've checked the WP core source code. The value is actually [translated](https://core.trac.wordpress.org/browser/trunk/src/wp-login.php#L1191), but here we don't translate it, so it **is** being tampered with. Thankfully, this value is not checked in the backend. But some 3rd-party plugins could use it. It would be good to retain the original value here, if possible.

Reply:
Oh. That's true!

I'll adjust the JS code to dynamically extract the field's name and value and pass it as is to the backend.